### PR TITLE
update view name and templete origin task 156

### PR DESCRIPTION
**Pull request by osbbarr1**
the folder was renamed origin.html
the name was renamed in urls.py
the test_origin_GET was created

thank you very much

Oscar Baquero 👍 :)

### CODE REVIEW

File: sources/urls.py [hunk 1]
```diff
@@ -9,5 +9,5 @@
 # general urls
 urlpatterns = [
     path('vehicles', vehicles, name="vehicles"),
-    path('origin', origin, name="origin"),
+    path('parcel', origin, name="parcel"),
```
Comment (diferpaho) [inline sources/urls.py:12]: change this part
```suggestion
    path('parcel', parcel, name="parcel"),
```

File: main/tests/test_views.py [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 class TestViews(TestCase):
 
-
-
-
+    def test_parcel_GET(self):
```
Comment (diferpaho) [inline main/tests/test_views.py:7]: the test_parcel_GET must be in sources / test and not in main / test, also test_origen_GET exists and is to change it to test_parcel_GET.

**Review by diferpaho — COMMENTED** [review]
please review the comments and make corrections

**Comment by osbbarr1** [comment]
Adjustments are made to the software. I am attentive to comments

**Comment by diferpaho** [comment]
It gives me problems to approve a closed pull request, please create the pull request with return to the changes to be able to approve it.

**Pull request closed without merging**